plugin/db/driver.go

```diff
@@ -117,14 +117,28 @@ type Table struct {
 	IndexList []Index
 }
 
+// InstanceMetadata is the metadata for an instance.
+type InstanceMetadata struct {
+	UserList     []User
+	DatabaseList []DatabaseMetadata
+}
+
+// DatabaseMetadata is the metadata for a database.
+type DatabaseMetadata struct {
```

Comment:
If we don't plan to have another struct to contain table info inside it. I would just call this Database

Otherwise, this can be called DatabaseMeta

Reply:
Done. Used DatabaseMeta.